API/JSON:API.md

```diff
@@ -0,0 +1,3 @@
+JSON:API is a specification for formatting JSON APIs. Its purpose is to take API format design out of the equation so that you can focus on delivering your API. Due to its structure, tooling can be built around the specification which handles the bulk of work regarding parsing requests and formatting responses.
+
+The specification can be found [here](https://jsonapi.org/). If your work will concern building or working with JSON:API APIs, read the specification to understand the concepts behind it.
```

Comment:
Should we add JSON:API gems we're using?

Reply:
would add blog links to stuff like
https://oozou.com/blog/standardizing-restful-json-apis-with-openapi-spec-53
or 
https://nordicapis.com/the-benefits-of-using-json-api/

I had developers (ok, one developer) who did not know how to read the official documentation and we needed to explain it to him in a not so "documenty" way.

Reply:
I like both ideas 👍 

But I'll add them as a todo in productive and once we have all chapters more or less approved and deployed, we can then see everything as a whole and have a clearer picture where to fill in additional info 🙂